src/main/java/com/atlassian/db/replica/spi/jdbc/JdbcUrl.java

```diff
@@ -0,0 +1,56 @@
+package com.atlassian.db.replica.spi.jdbc;
```

Comment:
Is it SPI or API?

Reply:
It should be `api` because we do not require the client to implement/extend this.
Will move it to the appropriate package.